docs/source/advanced.rst

```diff
@@ -319,13 +319,16 @@ To set the ``lynx`` output width to 400 characters, use this filter setup:
          width: 400
 
 
-Configuring how long browser jobs wait for pages to load
+Tweaking JavaScript output
 --------------------------------------------------------
 
-For browser jobs, you can configure how long the headless browser will wait
+Browser jobs might sometimes fail to produce the desired results when loading websites that contain JavaScript.
+If JavaScript refuses to run you may try setting ``useragent`` to the user agent of the current version of a commonly used web browser.
```

Comment:
Do JavaScript libraries check for user-agent often?

Reply:
I have no idea in general. But the first and only site with javascript that I am tracking so far, dhl tracking https://www.dhl.de/de/privatkunden/dhl-sendungsverfolgung.html, refuses to output tracking details with the default user agent.